R/lt_model_lq.R

```diff
@@ -105,13 +104,20 @@ lt_model_lq <- function(
     maxit = 200, 
     ...
 ) {
-    # check if an optional fitted_logquad is specified
+    # securing Sex input
+    Sex <- tolower(Sex)
+    if(Sex%in%c("total", "t")){
+        Sex <- "b"
+    }
+    if(!nchar(Sex)==1){
+        Sex <- substr(Sex,1,2)
+    }
+    
     
-    Sex <- match.arg(Sex)
+    # check if an optional fitted_logquad is specified
     
-    if(!is.null(fitted_logquad)){
-        fitted_logquad
-    } else {
+    if(is.null(fitted_logquad)){
+        
         if(Sex == "b"){
             fitted_logquad <- fitted_logquad_b
```

Comment:
I wonder if the warning about `fitted_logquad_b` not being visible would be overcome by using `data(fitted_logquad_b)`? Just a guess without googling.

Reply:
I'll check